# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../comedi-driver/inlining-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- comedi-0.7.69/comedi/drivers/cb_pcimdda.c~	2003-04-28 19:27:19.000000000 +0200
+++ comedi-0.7.69/comedi/drivers/cb_pcimdda.c	2005-01-26 15:42:03.008293000 +0100
@@ -209,7 +209,17 @@
 -----------------------------------------------------------------------------*/
 
 /* returns a maxdata value for a given n_bits */
-static inline lsampl_t figure_out_maxdata(int bits);
+static inline lsampl_t figure_out_maxdata(int bits)
+{
+    lsampl_t max = 0;
+    int i;
+
+    for (i = 0; i < bits; i++) {
+      max <<= 1;
+      max |=  1U;
+    }
+    return max;
+}
 
 /* 
  *  Probes for a supported device.
@@ -486,18 +496,6 @@
 }
 
 
-/* returns a maxdata value for a given n_bits */
-static inline lsampl_t figure_out_maxdata(int bits)
-{
-    lsampl_t max = 0;
-    int i;
-
-    for (i = 0; i < bits; i++) {
-      max <<= 1;
-      max |=  1U;
-    }
-    return max;
-}
 
 
 
